### PR TITLE
Selene Engineering tweaks and atmos remaster

### PR DESCRIPTION
**Pull request by Sadboysuss**
## About The Pull Request
Changed engineering a bit and fixed some issues. Makes engineering less cramped
I will include all changes in the changelog.
Screenshots:
image
image
image
image
## Why It's Good For The Game
Some features were missing from Selene like external air ports, cyborg charger, proper distro room. I've fixed that
## Changelog
:cl:
add: External air ports, atmos air refilling station, empty cans for atmos, cell charger in engineering, cyborg charger in engineering, proper distro for atmos, 
del: Removed partial atmos can in SM room since that's not how cans are made anymore
qol: added layer manifolds for gas supply to pure
fix: removed engineering access from engi sec outpost.
/:cl:

### CODE REVIEW

**Comment by Pepsilawn** [comment]
Screenshot_20240322-225239_Chrome

Why is this back is it even connected to anything

**Comment by Pepsilawn** [comment]
The streamlined atmos set up is counter productive, there are plenty of missing components such as Mix to Distro being non-existent

**Review by Pepsilawn — CHANGES_REQUESTED** [review]
See previous two comments

File: _maps/map_files/SeleneStation/SeleneStation.dmm [hunk 1]
```diff
@@ -36591,6 +36782,13 @@
 /obj/structure/sign/poster/official/high_class_martini,
 /turf/closed/wall,
 /area/station/maintenance/fore)
+"nkA" = (
+/obj/machinery/atmospherics/components/binary/pump{
+	dir = 8;
+	name = "Distro Stagsing to Filter"
```
Comment (Pepsilawn) [inline _maps/map_files/SeleneStation/SeleneStation.dmm:36788]: what does that mean

**Comment by Pepsilawn** [comment]
There's a rogue edit in ordnance and I'm not sure what you mean by an access issue regarding the sec office

File: _maps/map_files/SeleneStation/SeleneStation.dmm (hunk 1 above)
Comment (Sadboysuss) [reply _maps/map_files/SeleneStation/SeleneStation.dmm]: fixed

**Comment by Sadboysuss** [comment]
> The streamlined atmos set up is counter productive, there are plenty of missing components such as Mix to Distro being non-existent

expanded distro staging:
image

**Comment by Sadboysuss** [comment]
updated screenshots to reflect latest commit

**Comment by Pepsilawn** [comment]
A bit excessive with the black pipe going around the whole thing this pipeline already serves this purpose just fine

image

**Comment by Pepsilawn** [comment]
I'm quite cool with the atmos and SM room change, id like the portables being moved to the hallway change being reverted though, scrunching aux storage makes it look worse and isnt worth the change. One way to keep the portables outside would be to move all of tile storage two tiles to the left to take over the empty maintenance space and being able to throw them in the new 4 tiles that got freed up

image
like this not sure if the illustration is explanatory enough

the decaling above CE office needs to be fixed too as they were done with the portables in mind

**Review by Pepsilawn — APPROVED** [review]

**Comment by Sadboysuss** [comment]
hell yeah

**Pull request merged**